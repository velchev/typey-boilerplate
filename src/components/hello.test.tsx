import React from 'react';
import renderer from 'react-test-renderer';
import Hello from './hello';

test('Hello component should match the snapshot', () => {
  const component = renderer.create(<Hello />);

  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
