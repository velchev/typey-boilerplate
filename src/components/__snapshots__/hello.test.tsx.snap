// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello component should match the snapshot 1`] = `
<h2>
  Hello from Typescript and React!
</h2>
`;
